src/app/pages/Swap/form/index.tsx

```diff
@@ -164,8 +171,22 @@ export const SwapForm = memo<Props>(({ publicKeyHash }) => {
     [allUsdToTokenRates]
   );
 
+  const inputAssetPrice = useAssetFiatCurrencyPrice(inputValue.assetSlug ?? TEZ_TOKEN_SLUG, network.chainId);
```

Comment:
```suggestion
  const inputAssetPrice = useAssetFiatCurrencyPrice(inputValue.assetSlug ?? "", network.chainId);
```

why using tezos price as a fallback?